features/issues/components/filters/filters.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect, useCallback } from "react";
+import styled from "styled-components";
+import { Select, Option, Input } from "@features/ui";
+import { useFilters, IssueLevel, IssueStatus } from "@features/issues";
+import { useProjects } from "@features/projects";
+
+const Container = styled.div`
+  padding-bottom: 2rem;
+  display: flex;
+  gap: 3rem;
+  justify-content: flex-end;
+`;
+
+export function Filters() {
+  const { handleFilters } = useFilters();
+  const { data: projects } = useProjects();
+  const [inputValue, setInputValue] = useState("");
+  const [projectName, setProjectName] = useState<string | undefined>(undefined);
```

Comment:
I'm curious why you have the `projectName` state? It doesn't seem to be used anywhere

Reply:
you're right. It is not necessary and has been removed.